charts/openforms/templates/deployment.yaml

```diff
@@ -31,47 +31,7 @@ spec:
       {{- end }}
       serviceAccountName: {{ include "openforms.serviceAccountName" . }}
       securityContext:
-        {{- toYaml .Values.podSecurityContext | nindent 8 }}
-      {{- if and .Values.global.configuration.enabled .Values.configuration.enabled .Values.configuration.initContainer.enabled }}
```

Comment:
`.Values.configuration.initContainer` -> I think these values should be removed as they are no longer needed.